client/src/components/forms/MailingList/index.tsx

```diff
@@ -1,11 +1,141 @@
 import React from 'react';
+import /*type*/ { WithStyles, Theme } from '@material-ui/core/styles';
+import { withStyles, createStyles } from '@material-ui/core/styles';
+import { Button } from '@material-ui/core';
+import { useState } from 'react';
+import NavigateNextIcon from '@material-ui/icons/NavigateNext';
+import NavigateBeforeIcon from '@material-ui/icons/NavigateBefore';
+import EmailSlide from './slides/Email';
+import FeedbackSlide from './slides/Feedback';
+import SocialSlide from './slides/Social';
+import { EVENT } from './types';
```

Comment:
Storing KVP's of the events in an enum. Waiting for Danicia to give the list

Reply:
👍